django/db/migrations/executor.py

```diff
@@ -358,19 +361,29 @@ def should_skip_detecting_model(migration, model):
 
                 # Handle implicit many-to-many tables created by AddField.
                 if field.many_to_many:
-                    if field.remote_field.through._meta.db_table not in existing_table_names:
+                    _db_table = (
```

Comment:
Does the `if model._meta.db_table not in existing_table_names:` check at line 344/347 needs the same treatment? I'm surprised the tests pass without case folding there. Any idea why?

Also this might be a more readable

```python
through_db_table = field.remote_field.through._meta.db_table
if ignores_table_name_case:
    through_db_table = through_db_table.casefold()
```

Reply:
> Does the if model._meta.db_table not in existing_table_names: check at line 344/347 needs the same treatment? I'm surprised the tests pass without case folding there. Any idea why?

@charettes That was my fault, yes you are right, this line needs the same treatment.
Also, I've changed the test and added 
```
options={
                'db_table': 'migrations_fakeiNITIalmODEl',
            },
```
to the `fake_initial/__init__.py ` to cover this line in test.